### PR TITLE
Tripmode: update zap stanza

### PR DESCRIPTION
**Pull request by ondrejfuhrer**
I have updated the zap stanza for all the files that I found left after uninstalling.

After making all changes to the cask:

- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name

### CODE REVIEW

File: Casks/tripmode.rb [hunk 1]
```diff
@@ -12,5 +12,13 @@
 
   app 'TripMode.app'
 
-  zap trash: '~/Library/Preferences/ch.tripmode.TripMode.plist'
+  zap trash: [
+               '/Library/Application Support/Tripmode',
+               '/Library/LaunchDaemons/ch.tripmode.nke.TripMode.plist',
+               '/Library/LaunchDaemons/ch.tripmode.TripMode.HelperTool.plist',
```
Comment (commitay) [inline Casks/tripmode.rb]: `uninstall launchctl` for the two `LaunchDaemons`

File: Casks/tripmode.rb [hunk 2]
```diff
@@ -12,5 +12,13 @@
...
+               '/Library/PrivilegedHelperTools/ch.tripmode.TripMode.HelperTool',
```
Comment (commitay) [inline Casks/tripmode.rb]: `uninstall delete` for this

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/tripmode.rb [hunk 3]
```diff
@@ -12,11 +12,14 @@
 
   app 'TripMode.app'
 
+  uninstall launchctl: [
+                         '/Library/LaunchDaemons/ch.tripmode.nke.TripMode.plist',
+                         '/Library/LaunchDaemons/ch.tripmode.TripMode.HelperTool.plist',
```
Comment (commitay) [inline Casks/tripmode.rb]: Don't use the full path to the plist.

See [here](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md#uninstall-key-launchctl) for how to find the launchjob id.

**Comment by ondrejfuhrer** [comment]
I also added `uninstall signal:` stanza. I tested the `quit` for the app, but didn't work.

**Review by commitay — APPROVED** [review]

**Pull request merged**